src/ontology/bspo-edit.owl

```diff
@@ -959,6 +963,23 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#shorthand> <ht
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/BSPO_0015202> "aboral to"^^xsd:string)
 TransitiveObjectProperty(<http://purl.obolibrary.org/obo/BSPO_0015202>)
 
+# Object Property: <http://purl.obolibrary.org/obo/BSPO_0020001> (in central side of)
+
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/BSPO_0020001> "BSPO:0020001")
```

Comment:
This is not necessary. This can be added in Uberon (the xref to itself). This is.. confusing here!

Add instead:

- dcterms:contributor (you)
- [editors note](http://purl.obolibrary.org/obo/IAO_0000116) "This was migrated from IRIOFORIGINALRELTION in Uberon"

Reply:
Maybe this is something from OBO format? Because the properties here also have this.

Reply:
Should I add the axioms in Uberon here?

```
[Typedef]
id: in_central_side_of
name: in_central_side_of
def: "X in_central_side Y <=> if Y is subdivided into left and right portions around some median divisor, all parts of X are closer to the median divisor than the outermost lateral sides." [BSPO:cjm]
property_value: seeAlso BSPO:0000059
property_value: seeAlso https://github.com/obophenotype/uberon/wiki/Modeling-paired-structures-Design-Pattern xsd:anyURI
is_a: part_of ! part of
```

Reply:
yes this should be in BSPO, NOT Uberon!

Reply:
then need to have a release with the new object property to use in uberon, right?

Reply:
Ok, I'll wait for the new BSPO release to add the xref in uberon.